.cfmigrations.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
That is a great question. I believe they would have to define a `default` manually, and @elpete  can correct me if I'm wrong, because this config supercedes the environment variables.   I know when I tried to run it without a manager specified it tells me that no `default` is present.   We will want to add this to the release docs, as well.